ppb/systems/__init__.py

```diff
@@ -141,12 +152,13 @@ def __init__(self, time_step=0.016, **kwargs):
     def __enter__(self):
         self.start_time = time.time()
 
-    def activate(self, engine):
+    def on_idle(self, idle_event: events.Idle, signal):
         if self.last_tick is None:
             self.last_tick = time.time()
```

Comment:
Is `time.time()` the right thing?

Reply:
That's what it used to be. I can update that to monotonic.

Reply:
I know it's not strictly in-scope, but while you're here, could you?